agenta-cli/agenta/sdk/agenta_init.py

```diff
@@ -75,16 +75,16 @@ def init(
                 )
             else:
                 try:
-                    get_app_id = client.list_apps(app_name=app_name)
-                    app_id = get_app_id.app_id
+                    apps = client.list_apps(app_name=app_name)
+                    app_id = apps[0].app_id
```

Comment:
let's add a check here. the wrong exception would be raised here is app_name does not exist (len(apps)==0 then 79 would fail). I'd also add an assert len(apps)<=1 just to make sure

Reply:
Thank you for catching that!